include/megaapi.h

```diff
@@ -9975,6 +9982,30 @@ class MegaApi
          */
         void getPaymentId(MegaHandle productHandle, MegaHandle lastPublicHandle, MegaRequestListener *listener = NULL);
 
+        /**
+         * @brief Get the payment URL for an upgrade
+         *
+         * The associated request type with this request is MegaRequest::TYPE_GET_PAYMENT_ID
+         * Valid data in the MegaRequest object received on callbacks:
+         * - MegaRequest::getNodeHandle - Returns the handle of the product
```

Comment:
We should document the rest of the request params available on all callbacks (ph, pht, phts). Same for `submitPurchaseReceipt`